src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -756,6 +762,15 @@ function if_include_struct_item_value(structValue, name, options)
   return options.inverse(this);
 }
 
+// To be used to verify that things are actually arrays before trying to use
+// #each with them, since that silently treats non-arrays as empty arrays.
```

Comment:
Unexpected octothorpe: '#each'

Reply:
Or is that a language thing?

Reply:
`#each` is https://handlebarsjs.com/guide/builtin-helpers.html#each

Reply:
So yes, it's a ZAP template thing.